CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ project(OpenTissue VERSION 0.994
 if(EXISTS ${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
   include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
   conan_basic_setup(TARGETS NO_OUTPUT_DIRS)
-  set(Boost_USE_STATIC_LIBS ON)
+  # set(Boost_USE_STATIC_LIBS ON)
```

Comment:
Why keep out commented CMake code around?

Reply:
this was an oversight...

Reply:
actually, I fixed this on the second commit as well.